core/consensus/authority/impl/schedule_node.hpp

```diff
@@ -46,10 +46,21 @@ namespace kagome::authority {
     using NoAction = Empty;
 
     struct ScheduledChange {
-      SCALE_TIE(2);
```

Comment:
Does this work? [`operator>>(std::shared_ptr)`](https://github.com/soramitsu/scale-codec-cpp/blob/master/include/scale/scale_decoder_stream.hpp#L86-L92)